index.html

```diff
@@ -62,11 +62,12 @@
             </span>
           </li>
         </ul>
+        <img class="two-column" src="./assets/ readme/background.jpg" alt="Торговый центр «Две пятерки»" title="Торговый центр «Две пятерки»">
```

Comment:
Please take a look at source. It contains redundant space `"./assets/ readme/background.jpg"` 
It would be better to check how it works locally. Now the picture is gone.

Reply:
At the same time you're doing well that you added title and alt for img tag. It's corect simantically.

Reply:
The space is required here as image is located on https://github.com/reviakin/gorkogo55-landing/blob/master/assets/%20readme/background.jpg. Please note empty space at the beginning of the " readme" folder name.
Do you want me to rename both src for the image and the folder name?
I've checked locally that image was displayed. Please let me know if you have any issues.